NCS.DSS.Customer/AzureSearchDataSyncTrigger/CustomerSearchDataSyncTrigger.cs

```diff
@@ -1,91 +1,90 @@
-using Azure;
 using Azure.Search.Documents.Models;
-using DFC.Common.Standard.Logging;
 using Microsoft.Azure.Functions.Worker;
 using Microsoft.Extensions.Logging;
 using NCS.DSS.Customer.Helpers;
-using NCS.DSS.Customer.ReferenceData;
-using Document = Microsoft.Azure.Documents.Document;
+using System.Text.Json;
 
 namespace NCS.DSS.Customer.AzureSearchDataSyncTrigger
 {
     public class CustomerSearchDataSyncTrigger
     {
-        private readonly ILoggerHelper _loggerHelper;
-        private readonly ILogger<CustomerSearchDataSyncTrigger> _log;
-        public CustomerSearchDataSyncTrigger(ILoggerHelper loggerHelper, ILogger<CustomerSearchDataSyncTrigger> log)
+        private readonly ILogger<CustomerSearchDataSyncTrigger> _logger;
+        public CustomerSearchDataSyncTrigger( ILogger<CustomerSearchDataSyncTrigger> logger)
         {
-            _loggerHelper = loggerHelper;
-            _log = log;
+            _logger = logger;
         }
 
         [Function("SyncDataForCustomerSearchTrigger")]
-        public async Task Run(
+        public async Task RunAsync(
             [CosmosDBTrigger("customers", "customers", ConnectionStringSetting = "CustomerConnectionString",
                 LeaseCollectionName = "customers-leases", CreateLeaseCollectionIfNotExists = true)]
-            IReadOnlyList<Document> documents)
+            IReadOnlyList<Models.CustomerDocument> documents)
         {
-            var correlationId = Guid.NewGuid();
-
-            _loggerHelper.LogMethodEnter(_log);
+            _logger.LogInformation("{functionName} started",nameof(CustomerSearchDataSyncTrigger));
 
             var client = SearchHelper.GetSearchServiceClient();
 
-            _loggerHelper.LogInformationMessage(_log, correlationId, "get search service client");
+            _logger.LogInformation("get search service client");
 
 
-            _loggerHelper.LogInformationMessage(_log, correlationId, "get index client");
+            _logger.LogInformation("get index client");
 
-            _loggerHelper.LogInformationMessage(_log, correlationId, "Documents modified " + documents.Count);
+            _logger.LogInformation("Documents modified {count}",documents.Count);
```

Comment:
these 2 logs do not seem to relate to any code, can you clarify what they are here for